plugins/SegmentEditor/templates/_unprocessedSegmentMessage.twig

```diff
@@ -1,17 +1,22 @@
 {% set visitorLogLinkHtml %}<a href="{{ visitorLogLink }}" target="_blank">{% endset %}
-{% if isSegmentToPreprocess %}
-<p>
-    {{ 'SegmentEditor_UnprocessedSegmentNoData1'|translate('<strong>(' ~ segmentName ~ ')</strong>')|raw }}
-    {{ 'SegmentEditor_UnprocessedSegmentNoData2'|translate }}
-</p>
-{% else %}
-<p>
-    {{ 'SegmentEditor_CustomUnprocessedSegmentApiError1'|translate }}
-    {{ 'SegmentEditor_CustomUnprocessedSegmentNoData'|translate }}
-</p>
-{% endif %}
-<p>
-    {{ 'SegmentEditor_UnprocessedSegmentInVisitorLog1'|translate(visitorLogLinkHtml, '</a>')|raw }}
-    {{ 'SegmentEditor_UnprocessedSegmentInVisitorLog2'|translate }}
-    {{ 'SegmentEditor_UnprocessedSegmentInVisitorLog3'|translate }}
-</p>
\ No newline at end of file
+    {% if isSegmentToPreprocess %}
+        {% if not archived %}
+            <p>
+                {{ 'SegmentEditor_UnprocessedSegmentNoData3'|translate }}
```

Comment:
Does it make sense to show this message and also the others below? First it says the data is unavailable and the second part then says it will become available in a couple of hours maybe. That might irritate people...